tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2386,6 +2386,19 @@ public function userGetsAllTheSharesSharedWithHimUsingTheSharingApi(string $user
 		);
 	}
 
+  /**
+	 * @When user :user gets share :share shared with him using the sharing API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsTheShareSharedWithHimUsingTheSharingApi(string $user, $share):void {
+		$user = $this->getActualUsername($user);
+    $shareId = $this->getLastPublicLinkShareId();
```

Comment:
Public link id for group share?

Reply:
sorry, I didn't understand comment

Reply:
Sorry for my concise comment :smile: 
What I meant to say is that you are using this step in this group share scenario `Scenario: A user shares the folder to the group`. It feels a bit confusing to see `getLastPublicLinkShareId()` being used to get the shareId of that group share.

Reply:
- admin shares resource to user or group(don't matter) and sets shareId as value `$this->setLastShareData()`
- recipient user (he can be group member) gets shareId `getLastPublicLinkShareId()` and sends request GET with shareId `https://localhost:9200/ocs/v1.php/apps/files_sharing/api/v1/shares/shareID` and checks response.